proto/kvrpcpb.proto

```diff
@@ -655,3 +655,104 @@ message ReadIndexResponse{
     uint64 read_index = 2;
 }
 
+enum VerOp {
+    VerPut = 0;
+    VerDel = 1;
+}
+
+message VerMutation {
+    VerOp op = 1;
+    bytes key = 2;
+    bytes value = 3;
+}
+
+message VerValue {
+    bytes value = 1;
+    uint64 version = 2;
+}
+
+message VerKvPair {
+    KeyError error = 1;
+    bytes key = 2;
+    repeated VerValue values = 3;
+}
+
+message VerGetRequest {
+    Context context = 1;
+    bytes key = 2;
+    string cf = 3; 
+    int32 version_num = 4;
+    uint64 start_version = 5; // start_version == 0 means without start version
+}
+
+message VerGetResponse {
+    errorpb.Error region_error = 1;
+    string error = 2;
```

Comment:
make a VerKeyError type? Maybe for now it only contain a single string field.

Reply:
fixed it. PTAL again. @yiwu-arbug